libs/services/designer-client-services/src/lib/base/connector.ts

```diff
@@ -77,10 +79,10 @@ export abstract class BaseConnectorService implements IConnectorService {
     const configuration = await getConfiguration(connectionId ?? '');
 
     if (this._isClientSupportedOperation(connectorId, operationId)) {
-      if (!this.options.valuesClient[dynamicOperation]) {
+      if (!this.options.valuesClient?.[dynamicOperation]) {
```

Comment:
Can you make the schema and values client optional then?

Reply:
Yes I can do that